src/py42/services/detectionlists/departing_employee.py

```diff
@@ -14,7 +14,13 @@
 
 
 class DepartingEmployeeFilters(_DetectionListFilters, Choices):
-    """Constants available for filtering Departing Employee search results."""
+    """Constants available for filtering Departing Employee search results.
```

Comment:
Right now these `high_risk_employee` and `departing_employee` filter classes are being generated by sphinx both in the methoddocs/...employee.md files and also in methoddocs/constants.md now.  The other classes (`CaseStatus`, `TrustedActivityType`, etc) are only being generated on the constants page.  

Should I... 
- Leave the filter classes displayed on both, leave everything as is
- Only have them on the constants page
- Define the other enum ones on both the constants and their own service page.

Reply:
Its determined we'll define the classes on both pages, and add a note that points to them being accessible in the `constants` module